test/script/src/Eval.c

```diff
@@ -4063,8 +4063,6 @@ void Eval_multiline_string(void) {
         "Special characters }{\"\"'',\n"
     );
 
-    ecs_os_free(ptr->value);
```

Comment:
Notice that this is no longer required, and actually segfaults now, since the string is freed by the generated destructor for the runtime struct defined in this test

Reply:
LGTM